### ISSUE TITLE
grpcio-tools does not support protobuf v4

### ISSUE THREAD

**Issue by gitpushdashf** [grpc/grpc#29969]
Currently set to 3.12 and later.

https://github.com/grpc/grpc/blob/master/tools/distrib/python/grpcio_tools/setup.py#L293

Would be very nice to use with protobuf v3 or v4.

Thank you!

**Comment by mdantonio** [comment]
Hello 👋
Just as a confirmation, is this issue solved with 1.51.1?
I just done a fresh installation and got protobuf 4.21.12 as a dependency

```
Successfully installed grpcio-tools-1.51.1 protobuf-4.21.12
```

**Comment by gnossen** [comment]
Yes, this issue was resolved in `1.49.1`
